_layouts/default.html

```diff
@@ -213,5 +220,8 @@ <h2 class="text-delta">Table of contents</h2>
     </script>
   {% endif %}
   <script src="{{ '/assets/js/header-nav.js' | relative_url }}"></script>
+  {% if site.search_enabled != false and site.use_custom_search != false %}
```

Comment:
I was thinking if checking for `true` made more sense here?
Something like 

```
 {% if site.search_enabled and site.use_custom_search %}
```

Reply:
JTD defaults `search_enabled` to true and hence we are forced to use `== false` or `!= false` as the value of `false` is the only thing that turns it off.